client/cli/ok.py

```diff
@@ -92,6 +93,12 @@ def parse_input(command_input=None):
 
 def main():
     """Run all relevant aspects of ok.py."""
+
+    #Checking user's Python bit version
+    bit_v = (8 * struct.calcsize("P"))
+    if (bit_v == 32):
+        print("32 bit Python is not supported. Please install the 64 bit version.")
```

Comment:
It might be strange to a student if they see this message, but OK works anyway ("OK worked this time. Do I really need to update to 64 bit?").

Another concern is that a typical OK session prints a lot of output to the screen, so this message will probably be lost and rarely seen by the the student. I suggest exiting immediately if 32-bit Python is detected (`exit(1)`). I'm not familiar with the 32-bit issue though; @Sumukh , what do you think?